src/librustdoc/html/templates/sidebar.html

```diff
@@ -0,0 +1,19 @@
+{%- if !title.is_empty() -%}
+<h2 class="location"> {#- -#}
+    <a href="#">{{ title_prefix }}{{title}}</a> {#- -#}
+</h2>
+{%- endif -%}
+<div class="sidebar-elems"> {#- -#}
+    {%- if it.is_crate() -%}
```

Comment:
This should also be a field reference: `if is_crate_page`. That will allow us to get rid of the `it` field of `Sidebar` entirely. Getting rid of it is nice because that makes it possible to see by looking at the `Sidebar` type what information it carries. If `Sidebar` includes a `clean::Item` there is a huge world of information it _could_ carry, and it's hard to reason about from the `Sidebar` type alone.